PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+# Description
+This PR provides new functions to use your bitcoin wallet, they allows you to deposit, withdraw and get balance value
+
+## Checklist
```

Comment:
1. By default the checkbox elements should be unchecked
2. `Items done` can be checkbox as well, this helps to have like TODO list when you initiate the PR, and then mark items as done while you are moving.

It seems ok for now, but keep in mind the above 🙂